src/ConfigCatProvider.tsx

```diff
@@ -103,95 +109,83 @@ function serverContextNotSupported(): Error {
   );
 }
 
+function providerIdText(providerId?: string) {
+  return providerId ? ` with id="${providerId}"` : " without id attribute";
+}
+
+export function createConfigCatProviderError(methodName: string, providerId?: string): Error {
+  return Error(`${methodName} must be used in ConfigCatProvider${providerIdText(providerId)}!`);
+}
+
 class ConfigCatClientStub implements IConfigCatClient {
-  readonly isOffline = true;
+  readonly isOffline: boolean;
```

Comment:
Let's revert this change.

```suggestion
  readonly isOffline = true;
```